calvinpete/Glokeeper-backend
# #181851623 Document the APIs using Swagger

#### What does this PR do?
This PR is to document the APIs using swagger

#### Description of Task to be completed?
Document the APIs using swagger to create a concise reference manual containing all the information required to work with the APIs

#### How should this be manually tested?
- First things first, fetch this branch by running the command `git fetch ft-document-api-181851623` in the terminal.
- Check into this branch by running `git checkout ft-document-api-181851623`
- Activate your virtual environment by running `source venv/bin/activate`
- Install dependencies by running `pip install -r requirements.txt`
- Start the application by running `python manage.py runserver`
- Then view the documentation by navigating to this [link](http://localhost:8000/api/documentation/)

#### What are the relevant pivotal tracker stories?
[#181851623](https://www.pivotaltracker.com/story/show/181851623)

#### Screenshots (if appropriate)
<img width="1434" alt="Screenshot 2022-08-19 at 10 09 56" src="https://user-images.githubusercontent.com/36506567/185564318-b942318a-38fc-4ecd-838d-b84b60146c6d.png">
